README.md

```diff
@@ -269,6 +269,18 @@ When changing options after configuration, you may omit the argument `build` if
 
 You can have multiple build directories! So if you are regularly building firmware for multiple probes we would recommend keeping an individual build directory configured for each one.
 
+If you are working with PowerShell you may have some issue while trying to configure some options like the enabled target list `-Dtargets=cortexm,stm`:
+
+``` console
+PS C:\...\blackmagic\build> meson configure build -Dtargets=cortexm,stm ParserError: Line | 1 | meson configure build -Dtargets=cortexm,stm | ~ | Missing argument in parameter list. PS D:\...\blackmagic\build>
```

Comment:
Does this correctly format over multiple lines, or are there some missing new lines here?

Think it wants to be:
```console
PS C:\...\blackmagic\build> meson configure build -Dtargets=cortexm,stm
ParserError:
Line | 1 | meson configure build -Dtargets=cortexm,stm
     | ~ | Missing argument in parameter list.
PS D:\...\blackmagic\build>
```

(if we've understood how PS formats this output)

Reply:
You're right, I messed up the formatting. I'm not familiar with the formatting and don't have PS handy so I'll trust you got it right.